library/src/containers/Schemas/SchemaProperties.tsx

```diff
@@ -176,12 +203,124 @@ export const SchemaPropertiesComponent: React.FunctionComponent<Props> = ({
               must match {element.schema.content.pattern}
             </span>
           )}
+          {element.schema.content.uniqueItems && (
+            <span
```

Comment:
I know these "spans" can be extracted to a component but don't want to lose time here refactoring since the React component itself will need a refactor and rethink soon.